client/src/views/Models/Model.vue

```diff
@@ -37,10 +37,17 @@
               slot="right"
               :selected-view-id="selectedViewId"
               :views="views"
+              :layouts="layouts"
+              :selected-layout-id="selectedLayoutId"
               @view-change="onSetView"
+              @layout-change="onSetLayout"
             />
           </settings-bar>
-          <global-graph v-if="selectedModel" :data="selectedGraph" @node-click="onNodeClick"/>
+          <global-graph
```

Comment:
to avoid loading error in case of missing graph, can we add the `v-if="selectedGraph"` back.

Reply:
Done.